src/mixxxmainwindow.cpp

```diff
@@ -104,8 +104,7 @@ MixxxMainWindow::MixxxMainWindow(
           m_toolTipsCfg(mixxx::TooltipsPreference::TOOLTIPS_ON) {
     DEBUG_ASSERT(pApp);
     DEBUG_ASSERT(pCoreServices);
-    m_pCoreServices->initializeSettings();
-    m_pCoreServices->initializeKeyboard();
+    m_pCoreServices->preInitialize(pApp);
```

Comment:
Why does this need to happen in the mixxxmainwindow constructor? can't we just do it beforehand? I'm working on moving the init logic out of MixxxMainWindow in #4110 because QML won't use MixxxMainWindow but needs to be initialized, too.

Reply:
Yes, Done.